pkg/models/openpitrix/applications.go

```diff
@@ -308,19 +308,20 @@ func (c *applicationOperator) ListApps(conditions *params.Conditions, orderBy st
 		sort.Sort(HelmApplicationList(apps))
 	}
 
-	items := make([]interface{}, 0, limit)
+	totalCount := len(apps)
```

Comment:
why not using https://github.com/kubesphere/kubesphere/blob/master/pkg/apiserver/query/types.go#L39 to parse request parameters since there are already code to deal with pagination?

Reply:
The parameter of openpitrix is different from resource api, so I just use the function GetValidPagination.

Reply:
This part of the code will be refactored in the next version. This version is temporarily compatible with the old api